tests/unittest_pyreverse_diadefs.py

```diff
@@ -21,7 +21,7 @@
 
 import astroid
 import pytest
-from unittest_pyreverse_writer import Config, get_project
+from unittest_pyreverse_writer import _DEFAULTS, Config, get_project
```

Comment:
Shouldn't this be defined in ``conftest.py`` and used as a pytest fixtures?

Reply:
Yes, that would be a good idea to keep the dependencies of the test modules as minimal as possible. I will do that.